front/src/applications/stdcmV2/components/StdcmResults/StdcmResultsTable.tsx

```diff
@@ -120,7 +112,7 @@ const StcdmResultsTable = ({
           })}
         </tbody>
       </table>
-      <div className="display-all" style={!isSimulationSelected ? {} : { borderRadius: '0' }}>
+      <div className="display-all" style={!isSimulationRetained ? {} : { borderRadius: '0' }}>
```

Comment:
Should we avoid using `style` and add a class `simulation-retained` and use it with `cx()` ?
```suggestion
      <div className="display-all" style={!isSimulationRetained ? {} : { borderRadius: '0' }}>
```

Reply:
Done